apps/etterlatte-statistikk/src/main/kotlin/statistikk/database/RyddUtlandstilsnittDao.kt

```diff
@@ -56,7 +55,7 @@ class RyddUtlandstilsnittDao(
                 )
 
             if (utlandstilknytning == null) {
-                statement.setNull(1, NVARCHAR)
```

Comment:
Dette ville da fungert om du brukte `Types. NULL` i stedet for `Types.NVARCHAR`? 🤔 

Men setString null fungerer fjell også

Reply:
Hmm, har alltid trodd at `setNull` skal ha med seg kolonnetypen 🤔 men setString null feels good man :p

Reply:
Tipper det hadde fungert med `Types.VARCHAR` som du hadde tenkt også. 
Er `NVARCHAR` egentlig en greie i Postgres? 🤷‍♂️